nodogsplash2/files/nodogsplash.config

```diff
@@ -54,3 +60,8 @@ config nodogsplash
 
   #MAC addresses that do not need to authenticate
   #list trustedmac '00:00:C0:01:D0:1D'
+
+  # Set FW_MARK for compatibilty with other OpenWrt/LEDE Packages eg mwan3, sqm etc.
+  list fw_mark_authenticated '520'
+  list fw_mark_trusted '510'
+  list fw_mark_blocked '500'
```

Comment:
Has this fixed the problem with mwan3?

Reply:
This fixes one problem. I will be working on the other problem(s) later..